public/sections/todo/todo.html

```diff
@@ -0,0 +1,20 @@
+<ul class="collection with-header ">
+    <li class="collection-header center">
+        <h4><strong>TODO's List</strong></h4>
+        <input type="text" placeholder="Search" ng-model="findTodo">
+    </li>
+    <li class="collection-item" ng-repeat="todo in ctrl.todos | filter : findTodo">
+        <a><i class="material-icons waves-circle tiny " style="background : {{todo.color}}">circle</i></a>
```

Comment:
We should probably try to use the more specific CSS property `background-color`.